packages/inference/src/tasks/nlp/textGeneration.ts

```diff
@@ -14,6 +14,9 @@ export async function textGeneration(
 		...options,
 		taskHint: "text-generation",
 	});
+	if (typeof res === "object" && res.hasOwnProperty("choices")) {
+		return res;
+	}
```

Comment:
@coyotte508 , add this validation since, one can use textGeneration without stream, thus the response is OpenAI like, however I'm not sure how to address the types for this case

Reply:
[`5b06f7c` (#581)](https://github.com/huggingface/huggingface.js/pull/581/commits/5b06f7c6a4ab5218100a6b75de9348a18a591a14) ?